src/views/Viewer.vue

```diff
@@ -26,7 +26,7 @@
 		v-if="initiated || currentFile.modal"
 		id="viewer"
 		:class="{'icon-loading': !currentFile.loaded && !currentFile.failed}"
-		:clear-view-delay="isTesting ? -1 : 5000 /* prevent cypress timeouts */"
+		:clear-view-delay="isTesting ? -1 : isMobile ? -1 : 5000" /* prevent cypress timeouts and disable on mobile */
```

Comment:
@artonge does this work or did I choose the wrong syntax?

Reply:
```suggestion
 		:clear-view-delay="headerTimeout"
```
With this in the `computed` node:
```js
		/**
		 * Prevent cypress timeouts and disable on mobile.
		 * @returns {number}
		 */
		headerTimeout() {
			if (isTesting || isMobile) {
				return -1
			} else {
				return 5000
			}
```

OR

```suggestion
 		:clear-view-delay="isTesting || isMobile ? -1 : 5000" /* prevent cypress timeouts and disable on mobile */ 
```

Reply:
I go with the second one. Thanks for the suggestion :)